custom_components/google_home/manifest.json

```diff
@@ -1,10 +1,10 @@
 {
   "after_dependencies": ["cloud", "http", "zeroconf"],
-  "codeowners": ["@leikoilja", "@DurgNomis-drol", "@ArnyminerZ"],
+  "codeowners": ["@leikoilja"],
```

Comment:
Not that i have a need to be listed, but why are we removed? 😄

Reply:
haha, very fair point. 😄  
Thing is that there are more and more of us joining to contribute (in a meantime let's welcome @KapJI) and i don't know what is the right way of keeping that list up to date. There are multiple places where "codeowners" must be specified, like this and [this](https://github.com/leikoilja/ha-google-home/blob/2bc6b5482b0abcaeaefa7919a4dd7b7d25c9fdcb/pyproject.toml#L5) one. I am happy to add your handles at those places, but what's the right way of doing so? Shall we be adding @KapJI here as well soon? And if so, what is a measurable amount of contribution to add people to the list? 🤔 
You guys have any suggestions?

Reply:
Guideline by [HA docs](https://developers.home-assistant.io/docs/creating_integration_manifest/#code-owners):
> GitHub usernames or team names of people that are responsible for this integration. You should add at least your  GitHub username here, as well as anyone who helped you to write code that is being included.

I propose adding everyone here who has collaborator access to this repo.

And regarding `pyproject.toml`, we use it only for testing, let's keep only you there.

Reply:
You have some good points 🤔 Let's just leave it until we find a better way to credit/list maintainers.

Reply:
Sounds good with me :)